achievements/box.md

```diff
@@ -0,0 +1,45 @@
+# Box Achievements
```

Comment:
There is on ongoing discussion in #20 about capitalisation in titles. If we follow the EC style guide, only the first word takes a capital.